lib/task/digitalobject/digitalObjectRegenDerivativesTask.class.php

```diff
@@ -35,6 +35,13 @@ protected function configure()
       new sfCommandOption('connection', null, sfCommandOption::PARAMETER_REQUIRED, 'The connection name', 'propel'),
       new sfCommandOption('slug', 'l', sfCommandOption::PARAMETER_OPTIONAL, 'Information object slug', null),
       new sfCommandOption('type', 'd', sfCommandOption::PARAMETER_OPTIONAL, 'Derivative type ("reference" or "thumbnail")', null),
+      new sfCommandOption(
+        'media-type',
+        'm',
+        sfCommandOption::PARAMETER_OPTIONAL,
+        'Limit regenerating derivatives to a specific media type (e.g. "audio" or "image" or "text" or "video). "Other" is not supported',
+        null
+      ),
```

Comment:
I would say this is ok to run all on one line. I think it'd be good to have consistency in the formatting of these rows so that this new one matches the others in terms of formatting.

Reply:
:+1:  Addressed in https://github.com/artefactual/atom/pull/895/commits/f39f50030a35e12370d781a5eb8998c71920b6dc